docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -12,7 +12,7 @@ Requirements
 
 To connect to a ClickHouse server, you need:
 
-* ClickHouse (version 20.8 or higher) or Altinity (version 20.3 or higher).
+* ClickHouse (version 21.3 or higher) or Altinity (version 20.3 or higher).
```

Comment:
why did the Altinity version not change? Oversight?

Reply:
https://github.com/trinodb/trino/blob/master/plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java#L38